### ISSUE TITLE
Wildcard ignores

### ISSUE THREAD

**Issue by ahcox** [getgrav/grav-plugin-sitemap#12]
It would be useful to be able to do wildcard or regex ignores like:

```
/boring_chapter/*
*/boring/*
```

instead of 

```
/boring_chapter/page1
/boring_chapter/page2
/boring_chapter/page3
/chapter1/boring/page1
/chapter1/boring/page2
/chapter2/boring/page1
/chapter2/boring/page2
```

**Comment by NeonTigerWeb** [comment]
I agree - some sort of Inheritance would be useful - especially if it's wildcard based

**Comment by Gon3s** [comment]
Hi,

replace line 70 :
`if ($page->published() && $page->routable() && !preg_match("/(" . str_replace('/', '\\/', implode('|', $ignores)) .")/i", $page->route()))` 

And the sitemap.yaml:

`/boring_chapter/.*
.*/boring/.*`

K.

**Comment by flaviocopes** [comment]
@Gon3s you can submit a PR to include this for everyone

**Comment by Gon3s** [comment]
@flaviocopes Done here => https://github.com/getgrav/grav-plugin-sitemap/pull/19

**Comment by simonhaenisch** [comment]
I think this was fixed in #34 and can be closed?
